module-bsp/board/rt1051/bsp/eink/EinkDisplay.cpp

```diff
@@ -44,10 +43,79 @@ namespace service::eink
             settings.LUTDSize    = 0;
             return settings;
         }
+
+        unsigned int toWaveformTemperatureOffset(std::int32_t temperature) noexcept
+        {
+            if (temperature >= LUTTemperatureCritical)
```

Comment:
In a next PR: please place new lines or curly braces around these `if`s for readability.